server/datastore/mysql/software_titles.go

```diff
@@ -166,10 +166,11 @@ func (ds *Datastore) ListSoftwareTitles(
 				version = *title.VPPAppVersion
 			}
 			title.AppStoreApp = &fleet.SoftwarePackageOrApp{
-				AppStoreID:  *title.VPPAppAdamID,
-				Version:     version,
-				SelfService: title.VPPAppSelfService,
-				IconURL:     title.VPPAppIconURL,
+				AppStoreID:         *title.VPPAppAdamID,
+				Version:            version,
+				SelfService:        title.VPPAppSelfService,
+				IconURL:            title.VPPAppIconURL,
+				InstallDuringSetup: title.PackageInstallDuringSetup,
```

Comment:
Ah sorry I think I wasn't clear in our chat, this should be `InstallDuringSetup: title.VPPAppInstallDuringSetup`, where `VPPAppInstallDuringSetup` is a new column you return from the query (`vat.install_during_setup`).

Much like the `self_service` field, it exists both for software packages and VPP apps, so you'll need to rename the field in the query to `si.install_during_setup as package_install_during_setup` and add `vat.install_during_setup as vpp_app_install_during_setup`, and adjust the struct field accordingly:

```
PackageInstallDuringSetup *bool   `db:"package_install_during_setup"`
VPPAppInstallDuringSetup *bool `db:"vpp_app_install_during_setup"`
```

Reply:
thanks. I've made this update